### DISCUSSION TITLE
What about LI-FI ?

### DISCUSSION THREAD

**Discussion by TangerineDreamer** [pschatzmann/arduino-audio-tools#1191] [Q&A] [answered]
As i see,audiotools are involved in every way to transmit audio,
i'm thinking of a simple system infrared based,with a range of 10 -15 meters to transmit and receiving audio,but
using stereo coding emitter and decoding receiver,did you somehow make some investigation in that,to integrate
audiotools?

**Comment by pschatzmann** [comment, marked as answer]
The IR transmission rate using 38KHz modulation is typically around 2400 baud. 
This is even with a good codec much too slow for audio.

**Comment by TangerineDreamer** [comment]
That's a shame,i have an HIFI headphone with that infrared wireless,sounds great ,but i guess it's not really the
same transmission that is proposed in the arduino's world.
Some years ago,i played with  some DIY cheap modules from aliexpress,the sound was not so bad :)

**Comment by pschatzmann** [comment]
So, just give it a try! 
Since I do not have any of those, I just relied on what I found with Google...
